test/extended/util/http.go

```diff
@@ -0,0 +1,56 @@
+package util
+
+import (
+	"crypto/tls"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	o "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/kubernetes/test/e2e/framework"
```

Comment:
keep the same style.
```suggestion
	e2e "k8s.io/kubernetes/test/e2e/framework"
```